modules/formulize/admin/formindex.php

```diff
@@ -494,6 +507,78 @@ function patch40() {
             print "<p>If you have any questions about this upgrade issue, please contact <a href=mailto:formulize@freeformsolutions.ca>Freeform Solutions</a> for assistance.</p>";
         }
 
+        $screenpathname = XOOPS_ROOT_PATH."/modules/formulize/templates/screens/default/";
```

Comment:
This block already exists in formindex.php, with some changes. But I believe the current version should supersede the old one entirely, so this block can be removed.

Reply:
removed